### ISSUE TITLE
Getting VFX to attach to character movement?

### ISSUE THREAD

**Issue by WutHavUDun** [0ceal0t/Dalamud-VFXEditor#32]
Hello, I've only recently started getting into using this addon, but so far I've had a lot of fun learning the tools! That being said, I'm sorry if this is an obvious fix or has been asked already somewhere I didn't see.

Basically I have a 'blade swing' VFX attached to heavy swing that spawns on my character. While standing still the VFX appear to work just how I want. But if I cast the skill while moving, it just finishes the animation on the ground where I initially cast it (as if standing still) rather than moving along with my character. I've seen other VFX that will move with your character, so I looked all over for some setting that might control this, but haven't had any luck so far.

Any advice would be much appreciated! :)

**Comment by 0ceal0t** [comment]
check in the "binder" settings. Usually for these types of things, it's good to find a VFX which behaves like that (for example, the default heavy swing animation) and see how the binder settings are configured for that.

**Comment by WutHavUDun** [comment]
Thanks for the response! I had thought it might be a binder related thing. Unfortunately, there's no consistent difference between the binders on abilities that follow and abilities that are in fixed positions from what I can tell. I tried making the settings on my modded ability identical to an effect that behaved the way I wanted, and still no dice.

I also had the thought that the setting might be applied individually to each particle but there are a lot of particle settings and I don't really understand what most of them do so it's been a bit harder to test that.

**Comment by 0ceal0t** [comment]
I've had a little bit of time to look at this, and I'd suggest setting up binder "properties start" like this:
![image](https://user-images.githubusercontent.com/18051158/160026739-afecbc09-9183-497e-ae56-a0e53d353346.png)

also make sure that the binder is selected in the "Timelines" tab, both under the "parameters" and "items"
![image](https://user-images.githubusercontent.com/18051158/160026896-70c459a2-bef4-48f2-b002-7283d5deb6fb.png)

**Comment by WutHavUDun** [comment]
I was thankfully able to fix it, but not the way I thought. I tried copying the settings you provided (I believe I had already tried something similar before, but decided I may as well give it a second go). Still no dice. I ended up importing an entirely new binder from another ability and changed the settings to match those you provided, and sure enough, it worked correctly on the first try. 

Still not sure why the original binder wasn't behaving as expected, but glad to have found a solution either way. Thanks for taking the time to help me work through this. :)
